### ISSUE TITLE
Blobbers seem to be able to register twice

### ISSUE THREAD

**Issue by shravanshetty1** [0chain/0chain#80]
Doing `zbox ls-blobbers` on three.devnet at the time of creating this issue shows that the same blobber has been registered twice with different ids.

**Comment by guruhubb** [comment]
The best way to handle this is heath check which keeps only one id alive. Check the SC and see why the old blobber is still there.

**Comment by andrenerd** [comment]
@shravanshetty1 
could you clarify, by what blobber's properties do you identify the duplicated blobbers?

@guruhubb 
could we prevent duplicated registrations for blobbers, adding extra check right to the RegistrerBlobbers function:
https://github.com/0chain/blobber/blob/62b7249597dc44e9301880af083371ef4c7aed08/code/go/0chain.net/blobbercore/handler/protocol.go#L59
if so, what properties should be used to "compare" and filter out duplicates?

**Comment by guruhubb** [comment]
@andrenerd Yes, use blobber IP address, port, and public key ( and blobber ID). If any one of them are identical then disallow registration. blobber can choose to **update** registration with new IP address and port.

**Comment by andrenerd** [comment]
https://github.com/0chain/blobber/pull/109
